src/views/Profile.tsx

```diff
@@ -0,0 +1,21 @@
+import { useLoaderData } from 'react-router-dom';
+
+type Profile = {
+  id?: string;
+};
+
+type loaderProps = {
```

Comment:
[style] [non-blocking] generally speaking, types are conventionally defined in PascalCase, to avoid confusion and colisión with variables 


ESlint will help us with thi in the future.

Reply:
Fixed